src/main/java/com/twilio/models/OAuthRequestBody.java

```diff
@@ -0,0 +1,38 @@
+package com.twilio.models;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@ToString
+public class OAuthRequestBody {
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @JsonProperty("client_id")
+    @Setter
+    private String clientId;
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @JsonProperty("client_secret")
+    @Setter
+    private String clientSecret;
+
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    @JsonProperty("audience")
+    @Getter
+    @Setter
+    private String audience = "https://www.twilio.com/organizations";
```

Comment:
For testing purpose this has been fixed, We need to fetch audience and grantType from open api specs.

Reply:
I have added a PR in open-api with number 247